### ISSUE TITLE
Support multiple prompts in CompletionRequest

### ISSUE THREAD

**Issue by Code-Hex** [sashabaranov/go-openai#84]
Hi 👋 Thanks for developing a great library!

[Openai create-prompt documentation](https://platform.openai.com/docs/api-reference/completions/create#completions/create-prompt) says the `prompt` field is supported `encoded as a string, array of strings, array of tokens, or array of token arrays.` but this library seems to be supported only encoded as a string.

openai-js implemented type for that like this: https://github.com/threepointone/openai-js/blob/eaade749f2ead531d2ac9a2015184f7b6418a581/api.ts#L475

So I want to ask the question, How can I send a completion request with an array of string?

Thanks

**Comment by jamiesun** [comment]
[Use one API call to answer multiple questions](https://platform.openai.com/docs/models/2-use-one-api-call-to-answer-multiple-questions)

I think multiple questions should be supported, and this should be a way to maintain the context of the conversation

**Comment by vvatanabe** [comment]
@sashabaranov
This feature was implemented in PR #220. The requirements for this Issue have been met. Can you close this Issue?

Example:
```
CompletionRequest{
	Prompt:    []string{"Lorem ipsum", "Lorem ipsum", "Lorem ipsum"}, //use string or []string 
	Model:     GPT3Curie,
	MaxTokens: 5,
}
```

**Comment by vvatanabe** [comment]
Fixed #220
